### PR TITLE
Layout/jagrati/footer

### PR DESCRIPTION
**Pull request by Jagrati1213**
- Added footer component footer component

### CODE REVIEW

**Review by sid1213 — APPROVED** [review]

File: components/Footer/page.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import Style from "./style.module.scss";
+import Image from "next/image";
+import { Col, Row } from "antd";
+
+const style: React.CSSProperties = { background: "#0092ff", padding: "8px 0" };
```
Comment (sid1213) [inline components/Footer/page.tsx]: please use external style

File: components/Footer/page.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+const FooterComponent: React.FC = () => {
+  return (
+    <Row gutter={{ xs: 8, sm: 16, md: 24, lg: 32 }} className={Style.footer}>
+      <Col className="gutter-row">
```
Comment (sid1213) [inline components/Footer/page.tsx:9]: Please use Style from module.scss
use className={Style.classname}

Comment (sid1213) [reply components/Footer/page.tsx:9]: use this on entire component

**Review by sid1213 — CHANGES_REQUESTED** [review]

File: components/Footer/page.tsx (hunk 1 above)
Comment (Jagrati1213) [reply components/Footer/page.tsx]: Removed

File: components/Footer/page.tsx (hunk 2 above)
Comment (Jagrati1213) [reply components/Footer/page.tsx:9]: Because It comes with Ant design.

**Review by sid1213 — APPROVED** [review]

**Pull request merged**